src/assets/x.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" id="Ebene_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 600 600" style="enable-background:new 0 0 600 600;" xml:space="preserve">
+<path d="M454.3,575.45L295.64,416.88L145.63,567.1c-32.26,32.3-84.6,32.34-116.91,0.07
+	c-32.3-32.26-32.34-84.6-0.07-116.91l0,0l150.22-150.25L28.65,149.8C-3.61,117.5-3.58,65.16,28.73,32.89s84.64-32.23,116.91,0.07
+	l150.01,150.22L454.3,24.64c32.29-32.25,84.62-32.22,116.88,0.07c32.25,32.29,32.22,84.62-0.07,116.88L412.53,300.16l158.57,158.57
+	c32.39,32.15,32.58,84.46,0.43,116.85c-32.15,32.39-84.46,32.58-116.85,0.43c-0.14-0.14-0.28-0.28-0.42-0.42L454.3,575.45z"/>
+</svg>
```

Comment:
There are some properties in your SVG files which can be safely removed. Take a look at:
https://www.svgminify.com/
https://imageoptim.com/

Reply:
It looks like that you didn't format all your code. Take a look at this file.

Reply:
Thanks for the hint!